frontend/src/index.scss

```diff
@@ -0,0 +1,18 @@
+@import "../node_modules/bulma/bulma.sass";
```

Comment:
You can replace this import (and all the others) with
```diff
-@import "../node_modules/bulma/bulma.sass";
+@import "~bulma/bulma.sass";
```
(see https://facebook.github.io/create-react-app/docs/adding-a-sass-stylesheet)

Reply:
I have tried it and it did not work ! That's why I'm importing the stylesheets this way. Maybe I've done something wrong, don't know what thought.